tests/spark_config_test.py

```diff
@@ -1778,7 +1778,7 @@ def test_get_valid_jira_ticket(
         if expected_result:
             mock_log.info.assert_called_with(f'Valid Jira ticket provided: {jira_ticket}')
         else:
-            mock_log.warning.assert_called_with(f'Jira ticket missing or invalid format: {jira_ticket}')
+            mock_log.info.assert_called_with(f'Jira ticket missing or invalid format: {jira_ticket}')
```

Comment:
(feel free to ignore me since CI doesn't own the spark code y'all are changing)

fwiw: these kinds of assertions can be a bit painful down the line since they're essentially functioning as change detectors - for this and the other assertions below that are checking if a specific string was printed/exists: are they superfluous due to any other assertions being made? e.g., in this case: is asserting the result is as expected good enough?